solace-spring-cloud-stream-binder/solace-spring-cloud-stream-binder-core/src/main/java/com/solace/spring/cloud/stream/binder/health/contributors/FlowsHealthContributor.java

```diff
@@ -0,0 +1,35 @@
+package com.solace.spring.cloud.stream.binder.health.contributors;
+
+import com.solace.spring.cloud.stream.binder.health.indicators.FlowHealthIndicator;
+import org.springframework.boot.actuate.health.CompositeHealthContributor;
+import org.springframework.boot.actuate.health.HealthContributor;
+import org.springframework.boot.actuate.health.NamedContributor;
+
+import java.util.HashMap;
+import java.util.Iterator;
+import java.util.Map;
+
+public class FlowsHealthContributor implements CompositeHealthContributor {
+	private final Map<String, FlowHealthIndicator> flowHealthContributor = new HashMap<>();
+
+	public void addFlowContributor(String bindingName, FlowHealthIndicator flowHealthIndicator) {
```

Comment:
Flows aren't identified by a `bindingName`. They should be given IDs according to the pattern: `flow-<[0:concurrency)>`.

Example:
Suppose a consume binding has `concurrency=5`, then there would be 3 `FlowHealthIndicator`, identified as `flow-0`, `flow-1`, `flow-2`.